sprintQuovantis/sprintParam/views.py

```diff
@@ -103,3 +106,76 @@ def post(self, request):
                     "message": "Data not stored"
                 },
                 status=status.HTTP_400_BAD_REQUEST)
+
+
+class UserVote(APIView):
+    """
+    This class is creating for vote records
+    """
+    def post(self, request):
+        """
+        this method is created for inserting the voting data
+        :param request: format of the request
+        :return: Response
+        """
+        try:
+            serializer = VoteSerializer(data=request.data)
```

Comment:
Have you considered all validation scenarios? For example, a voter cannot vote for the same parameter in a sprint twice. But your code suggests that I can make a POST request twice and vote for the same parameter twice in the same sprint.